src/templates/script/parseScriptTemplatesV2.ts

```diff
@@ -70,38 +80,35 @@ interface IRawUserPrompt {
     enum?: {
         value: string;
         display: string;
-    };
+    }[];
     resource?: ResourceType,
     placeHolder?: string;
 }
 
-interface IParsedInputPrompt extends IRawUserPrompt {
-    assignTo: string;
-    defaultValue: string;
-    required: boolean;
+export interface ParsedInput extends RawUserPrompt, RawInput {
+
 }
 
-export interface IParsedJob extends IRawJob {
-    prompts: IParsedInputPrompt[];
-    // doesn't seem like there's a need to parse these yet
-    executes: IParsedAction[];
+export interface ParsedJob extends RawJob {
+    prompts: ParsedInput[];
+    executes: ParsedAction[];
```

Comment:
I'm not a huge fan of the mismatched property name and type names here. I know that each input will later correspond to a prompt step and each action an execute step. However, I don't love that the translation is done here in the property names before the inputs and actions are actually made into steps.

I'd rather have the property names match the types like so:
```suggestion
    parsedInputs: ParsedInput[];
    parsedActions: ParsedAction[];
```

I think this makes even more sense taking into account the prompt step factory definition is:
```ts
function promptStepFactory<T>(input: ParsedInput): AzureWizardPromptStep<T>;
```